### PR TITLE
Fix dependencies and imports

### PR DESCRIPTION
**Pull request by pp-mo**
Addresses the 'remaining work' noted in [this #1822 comment](https://github.com/SciTools/iris/issues/1822#issuecomment-155483860)

### CODE REVIEW

File: lib/iris/fileformats/rules.py [hunk 1]
```diff
@@ -121,7 +120,6 @@ def _rules_execution_environment():
         _rules_globals.update(iris.coords.__dict__)
         _rules_globals.update(iris.coord_systems.__dict__)
         _rules_globals.update(iris.fileformats.um_cf_map.__dict__)
-        _rules_globals.update(cf_units.__dict__)
```
Comment (marqh) [inline lib/iris/fileformats/rules.py]: I don't understand what this used to do and why it is now obsolete

Comment (pp-mo) [inline lib/iris/fileformats/rules.py]: It isn't.
I got confused...
I will fix that + force-upload it, to avoid having that code reversion in the branch...

**Comment by pp-mo** [comment]
Note: forced an upload, to hide the introduction+reversion of the mistake spotted by @marqh

**Comment by marqh** [comment]
that's fine, thanks @pp-mo

**Pull request merged**